decomp2gef.py

```diff
@@ -40,7 +41,20 @@ def rebase_addr(addr, up=False):
     down -> make an absolute address to an offset
     """
     vmmap = get_process_maps()
-    base_address = min([x.page_start for x in vmmap if x.path == get_filepath()])
+
+    if is_remote_debug():
```

Comment:
Since this, and the other code you have duplicates this logic please place it in a global function at the beginning of this file named `get_text_base_address`.